config/locales/es.yml

```diff
@@ -894,7 +894,7 @@ es:
         users: Usuarios
         zones: Zonas
       taxons:
-        display_order: Mostrar Pedido
+        display_order: "Orden de presentación"
```

Comment:
🤔 Not sure about this translation 👉 👈 

I preferred the other one.  But Spanish is so different depending of the region.